### ISSUE TITLE
Wishlist: Add support for synchronization of contacts

### ISSUE THREAD

**Issue by csolisr** [gggard/AndroidCaldavSyncAdapater#120]
Right now I've found no free software to synchronize the contacts between ownCloud and my phone. Do you think it's possible to add that functionality to your program?

**Comment by jkufner** [comment]
I would love to have that too.

**Comment by silentboatman** [comment]
Yes, that would be great! But maybe as a separate app.. AndroidCarddavSyncAdapter

**Comment by jkufner** [comment]
Well... not really. Most of the configuration is the same for both CalDAV and CardDAV. Only a type of collection is different. When i look at Google account (as user), it has many applications on the list to sync. Also, it will be easier to maintain one application. If AndroidCaldavSyncAdapater is well designed, it should be piece of cake.

And do not forget tasks! And journal! It is the same problem as contacts.

**Comment by timoberger** [comment]
i would like to have that too :)

i was already thinking about a vcard adapter. there is simply no time at the moment for this kind of project.
@jkufner you are right, most of the protocol is the same. but most of the work within the adapter is the translation between the vevent and the content provider.

**Comment by silentboatman** [comment]
@jkufner  Yes I suppose your right. If it got a lot of configuration in common, maybe it should be in the same app. Anyhow, it would be a great feature!
